crates/runtime/tests/models/hf.rs

```diff
@@ -123,19 +133,20 @@ mod nsql {
                     TestCase {
                         name: "hf_with_model",
                         body: json!({
-                            "query": "how many records (as 'total_records') are in taxi_trips dataset?",
+                            "query": "how many records (as 'total_records') are in spice.public.taxi_trips dataset?",
```

Comment:
Was unable to get model to use the right table schema w/o this change - to be addressed separately (model tried to use `catalog.spice.taxi_trips`).